src/HasMedia/MediaCollection.php

```diff
@@ -146,7 +142,7 @@ public function isImage() {
 
 	//FIXME: metoda disk by mohla mat druhy nepovinny paramater private, ktory len nastavi interny flag na true. Aby sme vedeli presnejsie ci ide o private alebo nie
 	public function isPrivate() {
-		return $this->disk == config( 'media-collections.private_disk' );
+		return $this->diskName == config( 'media-collections.private_disk' );
```

Comment:
This should be handled differently, somewhere there was a PR to solve this. In my opinion it would be better if the config of disks has the private info as property.